pyfolio/perf_attrib.py

```diff
@@ -105,6 +105,13 @@ def perf_attrib(returns, positions, factor_returns, factor_loadings,
     # cash will not be in factor_loadings
     num_stocks = len(positions.columns) - 1
     missing_stocks = missing_stocks.drop('cash')
+    num_stocks_covered = num_stocks - len(missing_stocks)
+
+    if num_stocks_covered == 0:
+        warnings.warn("Could not perform performance attribution. No factor "
+                      "loadings were available for this algorithm's "
+                      "positions.")
+        return None, None
```

Comment:
Do all the downstream consumers of this handle `None` return values?

Reply:
ah true, should I just have consumers return `None` too?

Reply:
I guess it depends on the context. Actually, should this just raise an exception? Doesn't seem like there's anything valuable we can do once we hit this case?

Reply:
I'm okay with that - @abhijeetkalyan had initially wanted a warning instead of raising an exception, you okay with raising an exception here?

Reply:
We discussed offline - going with an exception

Reply:
fixed